pkg/virt-launcher/virtwrap/network/nichotplug_test.go

```diff
@@ -17,7 +17,7 @@
  *
  */
 
-package virtwrap
```

Comment:
The `make generate` lane fails because a test suite file is missing from the newly added package.
Could you please add it? (See `pkg/network/multus/multus_suite_test.go` for example).